apps/tiltakskoordinator-flate/src/components/DeltakerDetaljerHeader.tsx

```diff
@@ -0,0 +1,54 @@
+import { BodyShort, Heading } from '@navikt/ds-react'
+import { DeltakerDetaljer } from '../api/data/deltaker'
+import { erAdresseBeskyttet, lagDeltakerNavn } from '../utils/utils'
+import { DeltakerMarkering } from './DeltakerMarkering'
+import { Fodselsnummer } from './Fodselsnummer'
+
+interface Props {
+  deltaker: DeltakerDetaljer | null
+}
+
+export const DeltakerDetaljerHeader = ({ deltaker }: Props) => {
+  if (!deltaker) {
+    return null
+  }
+
+  const adresseBeskyttet = erAdresseBeskyttet(deltaker.beskyttelsesmarkering)
+
+  return (
+    <div className="pt-4 pb-4 border-t border-b border-[var(--a-border-divider)]">
+      <div className="flex items-center flex-wrap gap-2 pl-10 pr-10">
+        <Heading level="2" size="small">
+          {lagDeltakerNavn(
+            deltaker.fornavn,
+            deltaker.mellomnavn,
+            deltaker.etternavn
+          )}
+        </Heading>
+        <HeaderSkille />
+
+        {!adresseBeskyttet && deltaker.fodselsnummer && (
```

Comment:
Backend sørger for tilgangssjekk på addressebeskyttelse fordi det er ikke mulig å gjøre det i frontend

Reply:
Har lagt inn sjekken i backend nå

Reply:
fikset